src/logs/index.js

```diff
@@ -0,0 +1,41 @@
+import Promise from 'bluebird'
+import AWS from '../util/aws'
+import { getLogGroup } from '../util/aws/cloudwatch-logs'
+import { getAliasVersion } from '../util/aws/lambda'
+import getLogs from '../util/get-logs'
+import { pkg } from '../util/load'
+import genName from '../util/generate-name'
+
+export default function (opts) {
+  const functionName = genName(opts.name).fullName
+  const aliasName = opts.stage
+  const stream = opts.stream
+  const region = opts.region || pkg().shep.region
+
+  AWS.config.update({ region })
+
+  // Should gracefully handle if function doesn't exist
+  return Promise.all([getLogGroup({ functionName }), getAliasVersion({ functionName, aliasName })])
```

Comment:
Check out `Promise.join` here. It basically does what you are doing with `all` and `spread` but in one command